render/render_test.go

```diff
@@ -11,8 +11,11 @@ import (
 	"html/template"
 	"net/http"
 	"net/http/httptest"
+	"strconv"
 	"testing"
 
+	"strings"
```

Comment:
remove the extra empty line between internal package.

Reply:
Done ;-)